sympy/core/relational.py

```diff
@@ -665,6 +661,26 @@ def _eval_simplify(self, **kwargs):
                 pass
         return e.canonical
 
+    def integrate(self, *args, **kwargs):
+        """See the integrate function in sympy.integrals"""
+        from sympy.integrals import integrate
+
+        # With indefinite integrals we can not conclude that
+        #   x = y  =>  Integral(x) = Integral(y)
+        # because there should be an arbitrary integration constant.
+        isdefinite = lambda a: isinstance(a, (tuple, Tuple)) and len(a) == 3
```

Comment:
The tuple `(x, a, b)` may have `None` for `a` and/or `b`. It seems that `None not in a` should be added.